docs/CHANGELOG.md

```diff
@@ -9,6 +9,26 @@ This project uses [*towncrier*](https://towncrier.readthedocs.io/) for keeping t
 
 <!-- towncrier release notes start -->
 
+# ## [1.7.0](https://github.com/pypa/pipx/tree/1.7.0) - 2024-08-19
```

Comment:
This seems wrong (and looks wrong in the rendered page as well). @chrysle do you reckon this is the workflow, or a bug in towncrier? :thinking:

Reply:
done